### DISCUSSION TITLE
Zen Beta Browser, Zen Dev Browser, Zen Nightly Browser

### DISCUSSION THREAD

**Discussion by BraveBrowserFan** [zen-browser/desktop#3976] [Ideas]
Hi

Why not have Zen Dev Browser, Zen Nightly Browser. Turn Zen Twilight browser into   Zen Twilight  Beta. thanks

**Comment by adminy** [comment]
I'd be happy if you have a latest or beta tag, so that I don't have to manually bump every time.
